addon/components/faceted-search/style.scss

```diff
@@ -0,0 +1,56 @@
+/* Override in consuming application to customize appearance of search facets */
+
+.panel-default {
+    font-family: 'Open Sans' , 'Helvetica Neue' , sans-serif;
+    -webkit-font-smoothing: antialiased ;
```

Comment:
OK - removed. I copied this style from Ember-SHARE and assume it was done intentionally.  I can't really tell the difference in the smoothness with or without it so I will remove it. I might be missing something.